modules/custom/az_gdpr_consent/js/az-gdpr-consent.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * @file
+ * AZ GDPR Consent Management - Pure JavaScript implementation (ES6)
+ *
+ * This script uses client-side geolocation detection via Pantheon's /cdn-loc
+ * endpoint to determine visitor location and conditionally auto-accepts
+ * consent for visitors from non-GDPR countries.
+ *
+ * For GDPR countries, the Klaro banner displays normally and toggle button is shown.
+ * For non-GDPR countries, consent is auto-accepted and toggle button stays hidden.
+ */
+
+/* eslint-disable no-console */
+
```

Comment:
See https://www.drupal.org/docs/drupal-apis/javascript-api/javascript-api-overview for more information